src/Api/Projects.php

```diff
@@ -1715,6 +1715,25 @@ public function deleteProjectAccessToken($project_id, $token_id)
         return $this->delete($this->getProjectPath($project_id, 'access_tokens/'.$token_id));
     }
 
+    /**
+     * @param string|int $project_id
+     * @param string|int $token_id
+     * @param string $expiry
+     *
+     * @return mixed
+     */
+    public function rotateProjectAccessToken(string|int $project_id, string|int $token_id, string $expiry = ''): mixed
```

Comment:
Crosslinking both (open) GitHub pull requests with the same method/functionality:
1. https://github.com/GitLabPHP/Client/pull/800/files#diff-160122af7affd3b85f6c02a3c1fcd7f20fbda356ab8bf3c89f71d0a56f68b31bR1725
2. https://github.com/GitLabPHP/Client/pull/808/files#diff-160122af7affd3b85f6c02a3c1fcd7f20fbda356ab8bf3c89f71d0a56f68b31bR1717